components/knative-function-controller/Makefile

```diff
@@ -18,20 +18,9 @@ else
 endif
 
 # Run tests
-test: generate fmt vet symlinks
+test: generate fmt vet
```

Comment:
Please revert to `dynamic generation of symlinks`. Introducing softlines caused a bug in a prow job. The symlinks can be resolved first when dependencies are pulled to vendor.

https://github.com/kyma-project/kyma/pull/5012/files